cloud/ec2.go

```diff
@@ -1307,16 +1310,16 @@ func (m *ec2Manager) DeleteVolume(ctx context.Context, volume *host.Volume) erro
 	return errors.Wrapf(volume.Remove(), "error deleting volume '%s' in db", volume.ID)
 }
 
-func (m *ec2Manager) extendVolumeExpiration(ctx context.Context, volume *host.Volume) error {
-	if err := volume.SetExpiration(time.Now().Add(evergreen.DefaultSpawnHostExpiration)); err != nil {
+func (m *ec2Manager) modifyVolumeExpiration(ctx context.Context, volume *host.Volume, newExpiration time.Time) error {
+	if err := volume.SetExpiration(newExpiration); err != nil {
 		return errors.Wrapf(err, "can't update expiration for volume '%s'", volume.ID)
 	}
 
 	_, err := m.client.CreateTags(ctx, &ec2.CreateTagsInput{
 		Resources: []*string{aws.String(volume.ID)},
 		Tags: []*ec2.Tag{{
 			Key:   aws.String(evergreen.TagExpireOn),
-			Value: aws.String(expireInDays(evergreen.SpawnHostExpireDays))}},
+			Value: aws.String(newExpiration.Add(evergreen.ExpireOnCushion).Format(evergreen.ExpireOnFormat))}},
```

Comment:
Why do we need an expire on cushion? Does the reaper not already handle this? I know for hosts we don't actually terminate for like a week or something

Reply:
my understanding of the reaper is that you get a grace period after the date in the expire-on tag. Nonetheless it makes sense to me to stay on its good side.
I think we do this for no-expire hosts too: when a host gets within 24 hours of its Evergreen expiration we bump its expire-on tag to [30 days in the future](https://github.com/evergreen-ci/evergreen/blob/43a34d693008bb069e6622a7541d228a43c36708/cloud/ec2.go#L658).

Reply:
Hmm so can we standardize this then--either using expireInDays with a constant in both places, or ExpireOnCushion?

Reply:
done